redirects.yaml

```diff
@@ -1,3 +1,16 @@
+# ledgerstatefix docs
+'/docs/ledger-state-fix':
```

Comment:
The single quotes here and on the next line are optional. I suggest removing them for consistency with how the other entries are now (after I removed them from everything).

Reply:
Updated to remove single quotes.